packit_service/models.py

```diff
@@ -149,13 +150,78 @@ def get_or_create(
                 session.add(project)
             return project
 
+    @classmethod
+    def get_projects_range(cls, first: int, last: int):
```

Comment:
I would say there is no need to includ the `_range` suffix here and in the name of the following method. Method names don't need to tell anything about what arguments they take, so it's fine to just have `get_projects` and `get_project_prs`.

Reply:
Let's also document the return type, pls :)

Reply:
Sure